packages/gatsby/src/utils/handle-flags.ts

```diff
@@ -14,6 +14,22 @@ const terminalLink = (text, url): string => {
   }
 }
 
+let enabledConfigFlags: Array<IFlag>
+let message = ``
+let unknownFlagMessage = ``
+
+export const getCachedFlagInfo = (): {
+  enabledConfigFlags: Array<IFlag>
+  unknownFlagMessage: string
+  message: string
+} => {
+  return {
```

Comment:
Should we throw if this is called _before_ we call `handleFlags` (or do something else to indicate that flags weren't processed yet)? That's only thing that can cause some weirdness I think

Reply:
👍  added